client/src/modules/templates/bhTransactionTypeSelect.tmpl.html

```diff
@@ -0,0 +1,29 @@
+<div ng-form="$ctrl.formName" bh-transaction-type-select ng-model-options="{ updateOn: 'default' }">
+  <div 
+    class="form-group"
+    ng-class="{ 'has-error' : $ctrl.validationTrigger && $ctrl.formName.transactionType.$invalid }">
+    <label class="control-label" translate>
+      {{ $ctrl.label }}
+    </label>
+
+    <ui-select 
+        multiple
+        name="transactionType" 
```

Comment:
This should probably read `transactionTypes` since it now allows multiple transaction types.